docs/en/siem/rule-details/trusted-developer-application-usage.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[trusted-developer-application-usage]]
+=== Trusted Developer Application Usage
+
+Identifies possibly suspicious activity using trusted Windows developer
+activity.
```

Comment:
trusted Windows developer utility program activity

Reply:
@randomuserid 
Thanks for noticing this. We need to improve it a bit more:

`Identifies possibly suspicious activity using trusted Windows developer utility program activity.`

Should it be:
`Identifies possibly suspicious activity using a trusted Windows developer utility program.`?

Reply:
yes that is better